data.sql

```diff
@@ -29,4 +29,38 @@ UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
 UPDATE animals SET owner_id = 2 WHERE name = 'Gabumon' OR name = 'Pikachu';
 UPDATE animals SET owner_id = 3 WHERE name = 'Devimon' OR name = 'Plantmon';
 UPDATE animals SET owner_id = 4 WHERE name = 'Charmander' OR name = 'Squirtle' OR name = 'Blossom';
-UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
\ No newline at end of file
+UPDATE animals SET owner_id = 5 WHERE name = 'Angemon' OR name = 'Boarmon';
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, 'Apr 23, 2000'),
+('Maisy Smith', 26, 'Jan 17, 2019'),
+('Stephanie Mendez', 64, 'May 04, 1981'),
```

Comment:
- [ ] Kindly be noticed that the date of graduation for Stephanie Mendez is `May 17, 1981`.